pkg/pool-manager/pool.go

```diff
@@ -93,7 +94,7 @@ func NewPoolManager(kubeClient kubernetes.Interface, rangeStart, rangeEnd net.Ha
 	}
 
 	if kubevirtExist {
-		go poolManger.vmWaitingCleanupLook(waitTime)
+		go poolManger.vmWaitingCleanupLook(waitTime, mgr)
```

Comment:
store mgr as field of the PoolManager so we don't have to pass it to function.

Reply:
oh now I see what you mean. ack.